packages/realm-react/src/__tests__/useQueryHook.test.tsx

```diff
@@ -69,20 +52,22 @@ const testDataSet = [
   { _id: 6, name: "Sadie", color: "gold", gender: "female", age: 5 },
 ];
 
-describe("useQueryHook", () => {
+describe("useQuery", () => {
+  const context = createRealmTestContext({ schema: [dogSchema] });
```

Comment:
Similar comment as before (could destructure this here instead if you prefer).

Reply:
I don't think I can, because getting the `realm` from `context` needs to happen after it's been opened.